tests-behat/bootstrap/Context.php

```diff
@@ -380,7 +380,7 @@ public function iSelectValueInLookup($arg1, $arg2)
         $script = '$("#' . $lookup->getAttribute('id') . '").dropdown("show")';
         $this->getSession()->executeScript($script);
         // Wait till dropdown is visible
-        // Cannot call jqueryWait because calling it will return prior from dropdown to fire ajax request.
+        $this->jqueryWait();
```

Comment:
removed comment no longer valid - `jqueryWait` checks multiple times, thus AJAX should be already fired at that time